assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -5,61 +5,18 @@ import classNames from 'classnames';
 import { _n, sprintf } from '@wordpress/i18n';
 import { decodeEntities } from '@wordpress/html-entities';
 import type { ReactElement } from 'react';
-import type { PackageRateOption } from '@woocommerce/type-defs/shipping';
 import { Panel } from '@woocommerce/blocks-checkout';
 import Label from '@woocommerce/base-components/label';
-import { useSelectShippingRate } from '@woocommerce/base-context/hooks';
-import type { CartShippingPackageShippingRate } from '@woocommerce/type-defs/cart';
+import { useShippingData } from '@woocommerce/base-context/hooks';
```

Comment:
I merged the 2 hooks because they referenced each other and caused circular dependency.

Reply:
That's good 👍🏼